pbjson-test/src/lib.rs

```diff
@@ -427,6 +427,34 @@ mod tests {
 
```

Comment:
I'm wondering if checking in the generated file for the test setup would be helpful, because reviewing a bunch of `writeln!` calls is a bit tricky (I have to admit that I have about zero mental context for each of the writes).

Reply:
The difficulty is we compile the tests with various different feature flags, to test various different builder options, so I'm not really sure how to achieve this without checking in 5 different versions of the code...